spec/rails_helper.rb

```diff
@@ -35,6 +35,7 @@
   # examples within a transaction, remove the following line or assign false
   # instead of true.
   config.use_transactional_fixtures = true
+  Theblog.account_model = 'Incarnator::Account'
```

Comment:
Probably before
or create tests for different environments (with/without Incarnator)